website/docusaurus.config.js

```diff
@@ -132,54 +134,55 @@ const config = {
             items: [
               {
                 label: "Offchain Labs",
-                to: "https://offchainlabs.com/"
+                to: "https://offchainlabs.com/",
               },
               {
                 label: "Arbitrum.io",
-                to: "https://arbitrum.io/"
+                to: "https://arbitrum.io/",
               },
               {
                 label: "Nova.arbitrum.io",
-                to: "https://nova.arbitrum.io/"
-              }
-            ]
+                to: "https://nova.arbitrum.io/",
+              },
+            ],
           },
           {
             title: "Community",
             items: [
               {
                 label: "Discord",
-                href: "https://discord.gg/ZpZuw7p"
+                href: "https://discord.gg/ZpZuw7p",
               },
               {
                 label: "Twitter",
-                href: "https://twitter.com/ArbitrumDevs"
+                href: "https://twitter.com/ArbitrumDevs",
               },
               {
                 label: "Research forum",
-                href: "https://research.arbitrum.io/"
+                href: "https://research.arbitrum.io/",
               },
               {
                 label: "Medium Blog",
-                href: "https://medium.com/offchainlabs"
-              }
-            ]
-          }
+                href: "https://medium.com/offchainlabs",
+              },
+            ],
+          },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Offchain Labs, Built with Docusaurus.`
+        copyright: `Copyright © ${new Date().getFullYear()} Offchain Labs, Built with Docusaurus.`,
       },
       prism: {
         theme: lightCodeTheme,
-        darkTheme: darkCodeTheme
+        darkTheme: darkCodeTheme,
+        additionalLanguages: ["solidity"],
       },
       announcementBar: {
         id: "support_us",
         content: `Note: Arbitrum is in mainnet beta, which includes trusted admin controls; use at your own risk! See <a rel="noopener noreferrer" href="/mainnet-beta">here</a> for details.`,
         backgroundColor: "rgb(121 241 3)",
         textColor: "#091E42",
-        isCloseable: true
-      }
-    })
+        isCloseable: true,
```

Comment:
what's applying all these formatting/linting changes? If we're going to do a repo wide lint, can we include whatever it is into the package json for consistency? (eslint etc.)

Reply:
it's my local prettier
can have a new PR for prettier